.ci/main.yml

```diff
@@ -88,6 +88,13 @@ steps:
     packagesToPush: 'Output\nugets\UiPath.FreeRdpClient.*.nupkg'
     publishVstsFeed: '5b98d55c-1b14-4a03-893f-7a59746f1246/788028a9-5a01-48ee-b925-3af51ae46294'
 
+- task: NuGetCommand@2
```

Comment:
maybe just extentd the filter above to Uipath.*.nupkg

Reply:
I would also separate CI from CD. What do you think? Right now I'm wondering if the tests would fail on the build agent.

Reply:
also add/update step to run the tools tests